package.json

```diff
@@ -30,8 +30,8 @@
   "dependencies": {
     "webex": "^2.54.0",
     "@webex/component-adapter-interfaces": "^1.30.5",
-    "@webex/components": "^1.272.2",
-    "@webex/sdk-component-adapter": "^1.112.0"
+    "@webex/components": "^1.272.3",
```

Comment:
@rarajes2, why we don't have package-lock.json changes as part of upgrade dependencies

Reply:
Added `package-lock.json` file. Thanks